src/update-manager.ts

```diff
@@ -163,6 +171,13 @@ export default class UpdateManager {
               }
             }
           }
+
+          // honor wac-allow headers if present in non-http request
+          if ((uri as string).slice(0, 4) != 'http') {
```

Comment:
**suggestion**: Extracting the condition to a function like `isHttpUri(uri)` would make the code better readable